src/ui/Theme.h

```diff
@@ -11,7 +11,8 @@ namespace ui {
 enum class AvatarType
 {
         Image,
-        Letter
+        Letter,
+        Jdenticon
 };
```

Comment:
I think you can just delete this?

Reply:
Probably I forgot about that struct when I originally deleted the Avatar C++ stuff

Reply:
I forgot about it too, but I guess we can just delete it in this PR now :3